src/js/filter.js

```diff
@@ -168,7 +167,7 @@ class Filter {
     this.$controls.innerHTML = `
       <div class="select-container">
         <label for="sort-by"><span data-translation-id="sort_by">Sort by</span>: </label>
-        <select name="sort-by" id="sort-by" data-translate-font>
+        <select class="custom-select" name="sort-by" id="sort-by" data-translate-font>
```

Comment:
Not in love with `custom-select` as a class name, open to something more explicit if that would be preferred